data/README.md

```diff
@@ -67,6 +76,10 @@ instance, they should be in the same format as the files below.
   where each row represents a gene set, except this file does not include a
   description column, which is the second column in the GMT file format.
 
+  As we mentioned in the description of *sample_signature_activity.txt*,
+  depending on your preference of the signatures' order shown online, you may
+  need to pad some signatures' names.
```

Comment:
Slightly easier to read if we cut out some extra words and order the phrases this way:

```
As mentioned in *sample_signature_activity.txt* above, you may wish to pad numbers
in signature names so they will be sorted in a useful way.
```

Reply:
How about `... in a more useful way`?

Reply:
My English teacher would have pressed me on this: more useful than *what*? If we aren't going into detail on that, I think it makes sense to caution our readers to ensure that an alphabetical sort will produce something useful.

Reply:
Should we include the fact that we have added this padding to these two files available in this repository (in this section and the section for *sample_signature_activity.txt* above)?

Reply:
Probably not, for two reasons:
(1) These new instructions are for users who want to load their own data, not users who will be using our data (such as Hogan Lab);
(2) The demo page already shows that we have padded our signatures.